src/Workspaces/Core/Portable/Log/FunctionId.cs

```diff
@@ -345,5 +345,10 @@ internal enum FunctionId
         AssetService_GetAssetAsync,
         SnapshotService_RequestAssetAsync,
         CompilationService_GetCompilationAsync,
+        RemoteHostService_Synchronize,
+        AssetService_TryGetAsset,
+        AssetService_SynchronizeAssetsAsync,
+        AssetService_SynchronizeSolutionAssetsAsync,
```

Comment:
solution asset gets checksum of solution. from there it figure out all checksums missing to re-construct solution, and use SynchronizeAssetAsync to get all those missing assets.